src/__screenshot_tests__/list-screenshot-test.tsx

```diff
@@ -112,6 +112,7 @@ test('Rows with only a Title content are centered', async () => {
             subtitle: '',
             headline: '',
             description: '',
+            control: 'chevron',
```

Comment:
for some reason, the new storybook version doesn't use the chevron as default value for the control arg if not specified and the screenshots were failing